app/controllers/api/account.php

```diff
@@ -1582,13 +1590,20 @@
             $history = $user->getAttribute('passwordHistory', []);
             $validator = new PasswordHistory($history, $user->getAttribute('hash'), $user->getAttribute('hashOptions'));
             if (!$validator->isValid($password)) {
-                throw new Exception(Exception::USER_PASSWORD_RECENTLY_USED, 'The password was recently used', 409);
+                throw new Exception(Exception::USER_PASSWORD_RECENTLY_USED);
```

Comment:
@lohanidamodar I need your comments here. I saw that we return a 409 response, whereas this is more of a 400 error caused by a bad input. What do you think ?

Reply:
It should be okay 👌